src/_blog/js-libp2p-v1.md

```diff
@@ -0,0 +1,139 @@
+---
+tags:
+- browser
+- transport
+- webrtc
+- js-libp2p
+title: Announcing the release of js-libp2p v1.0.0 🎉
+description:
+date: 2023-11-22
+permalink: "/2023-11-22-js-libp2p/"
+translationKey: ''
+header_image: ''
```

Comment:
@2color any ideas for a header image?

Reply:
I'll try to whip something together quickly by altering one of the existing/older ones on Figma